lib/sensu/extensions/occurrences.rb

```diff
@@ -21,11 +38,11 @@ def description
       # @return [Array] containing filter output and status.
       def event_filtered?(event)
         check = event[:check]
-        occurrences = check[:occurrences] || 1
-        refresh = check[:refresh] || 1800
+        occurrences = str2int(check[:occurrences]) || 1
+        refresh = str2int(check[:refresh]) || 1800
```

Comment:
I see, I validated this:
```
$ irb
irb(main):001:0> a = 1
=> 1
irb(main):002:0> b = '1'
=> "1"
irb(main):003:0> a.to_i
=> 1
irb(main):004:0> b.to_i
=> 1
irb(main):005:0> nil.to_i
=> 0
```

I wonder if we should have a helper gem to have this as it could be used outside of this extenstion.